docs/conceptual/runtime/frame.md

```diff
@@ -44,47 +41,47 @@ It also has a number of system critical storage items, such as:
 Finally, it defines a number of low level functions which can access your blockchain storage, verify
 the origin of an extrinsic, and more.
 
-### Executive Module
+### Executive Pallet
 
-The [Executive module](https://substrate.dev/rustdocs/master/pallet_executive/index.html) acts as the orchestration layer
-for the runtime. It dispatches incoming extrinsic calls to the respective modules in the runtime.
+The [Executive pallet](https://substrate.dev/rustdocs/master/pallet_executive/index.html) acts as the orchestration layer
+for the runtime. It dispatches incoming extrinsic calls to the respective pallets in the runtime.
 
 ### Support Library
 
-The [SRML support library](https://substrate.dev/rustdocs/master/frame_support/index.html) is a collection of Rust macros,
-types, traits, and functions that simplify the development of Substrate runtime modules.
+The [FRAME support library](https://substrate.dev/rustdocs/master/frame_support/index.html) is a collection of Rust macros,
+types, traits, and functions that simplify the development of Substrate pallets.
 
 The support macros expand at compile time to generate code which is used by the runtime and reduce
-boilerplate code for the most common components of a module.
+boilerplate code for the most common components of a pallet.
 
 ### Runtime
 
-The runtime library brings together all these components and modules. It defines which modules are
+The runtime library brings together all these components and pallets. It defines which pallets are
 included with your runtime and configures them to work together to compose your final runtime. When
-calls are made to your runtime, it uses the Executive module to dispatch those calls to the
-individual runtime modules.
+calls are made to your runtime, it uses the Executive pallet to dispatch those calls to the
+individual pallets.
 
-## SRML Modules
+## FRAME Pallets
 
-The SRML contains a set of prebuilt runtime modules that can be used in your Substrate chain.
+FRAME contains a set of prebuilt pallets that can be used in your Substrate chain.
```

Comment:
I think this is not the right use of jargon. Fred was saying in riot that "frame" refers to the framework where we compose runtimes out of pallets. Frame does not refer to the collection of pallets that live in paritytech/substrate.

Here's a suggestion to write around the issue, but this choice is totally up to you.

```
## Prebuilt Pallets

Some pallets will be sufficiently general-purpose to be reused in many blockchains. Anyone is free to write and share useful pallets. There is a collection of popular pallets provided with Substrate. Let's explore them.
```

Reply:
Right. I'll take your suggestion.

For other references it will depend on the framework definition that we want to use. It could or could not contain pallets. 
>A framework is usually a library or some other piece of software that provides a skeleton for your application. It provides generic things that almost all applications need.

>A framework is a basic conceptual structure used to solve or address complex issues, usually a set of tools, materials or components.

>A framework is a particular set of rules, ideas, or beliefs which you use in order to deal with problems or to decide what to do.